.github/workflows/ci.yml

```diff
@@ -198,13 +200,26 @@ jobs:
           git config --global user.name "hikari-bot"
           git config --global user.email "90276125+hikari-bot[bot]@users.noreply.github.com"
 
+      - name: Temporarily disable "include administrators" branch protection
+        uses: benjefferies/branch-protection-bot@master
+        with:
+          access_token: ${{ secrets.PAT_TOKEN }}
+          enforce_admins: false
+
       - name: Deploy
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
           VERSION: ${{ github.event.release.tag_name }}
           REF: ${{ github.sha }}
           GITHUB_REPO_SLUG: ${{ github.repository }}
           DEPLOY_WEBHOOK_URL: ${{ secrets.DEPLOY_WEBHOOK_URL }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
         run: bash scripts/deploy.sh
+
+      - name: Re-enable "include administrators" branch protection
+        uses: benjefferies/branch-protection-bot@master
+        if: always()
+        with:
+          access_token: ${{ secrets.PAT_TOKEN }}
```

Comment:
random question, what does "PAT" mean in this context

Reply:
Public Access Token :)